cypress/e2e/filter/filterCases.cy.ts

```diff
@@ -136,6 +136,99 @@ describe("Case list", () => {
       cy.get("tr").not(":first").get("td:nth-child(1)").contains("Alfred Pennyworth")
     })
 
+    it("Should display cases filtered by court name", () => {
+      cy.task("insertCourtCasesWithFieldOverrides", {
+        keywords: { courtNames: ["Manchester Court", "London Court", "Bristol Court"] },
+        force: "011111"
+      })
+
+      cy.visit("/bichard")
+
+      cy.get("button[id=filter-button]").click()
+      cy.get("input[id=court-name]").type("Manchester Court")
+
+      cy.get("button[id=search]").click()
+      cy.get("tr").not(":first").get("td:nth-child(3)").contains("Manchester Court")
+      cy.get("tr").not(":first").get("td:nth-child(3)").contains("London Court").should("not.exist")
+      cy.get("tr").not(":first").get("td:nth-child(3)").contains("Bristol Court").should("not.exist")
+      cy.get("tr").should("have.length", 2)
+      cy.get(".moj-filter-tags a.moj-filter__tag").contains("Manchester Court")
+
+      // Removing filter tag
+      cy.get(".moj-filter-tags a.moj-filter__tag").contains("Manchester Court").click({ force: true })
```

Comment:
This is used a few times across the file. We could create a function for this that would take an argument (`Manchester Court`) in this case.